diff --git a/misc/CMakeLists.txt b/misc/CMakeLists.txt
index 2a5046ac..d5dd7a7d 100644
--- a/misc/CMakeLists.txt
+++ b/misc/CMakeLists.txt
@@ -50,7 +50,8 @@ configure_files(
   share/mime/packages/vnd.linyaps.uab.xml
   share/polkit-1/actions/org.deepin.linglong.PackageManager1.policy
   share/icons/linyaps.svg
-  share/applications/linyaps.desktop)
+  share/applications/linyaps.desktop
+  etc/linglong/config.json)
 
 # bash-completion
 set(BASH_COMPLETIONS_DIR ${CMAKE_INSTALL_DATADIR}/bash-completion)
@@ -178,6 +179,10 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/share/icons/linyaps.svg
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/script/linglong.sh
         DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/profile.d)
 
+# install systemd config
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/etc/linglong/config.json
+        DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/linglong)
+
 install(
   FILES ${CMAKE_CURRENT_BINARY_DIR}/script/linglong.sh
   DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/X11/Xsession.d/
diff --git a/misc/etc/linglong/config.json b/misc/etc/linglong/config.json
new file mode 100644
index 00000000..b85829c5
--- /dev/null
+++ b/misc/etc/linglong/config.json
@@ -0,0 +1,5 @@
+{
+    "device_mode": [
+        "passthru"
+    ]
+}
